clang/tools/dpct/DpctOptRules/cmake_script_migration_rule.yaml

```diff
@@ -2405,6 +2405,19 @@
       Out: ""
       RuleId: "remove_target_link_options_cuda_arg"
 
+#'-static-libstdc++' is not supported together with '-fsycl' when using icpx compiler.
```

Comment:
please confirm with compiler team, if it is correct or not.

Reply:
ok